mlflow/tensorflow.py

```diff
@@ -55,6 +55,7 @@
 
 # For tracking if the run was started by autologging.
 _AUTO_END_RUN = False
```

Comment:
Could we just remove this global & check whether `_AUTOLOG_RUN_ID` is `None` in our logic below?

Reply:
Yup, now that you mention it I see now I can refactor it.